README.md

```diff
@@ -536,6 +536,39 @@ Returns an array of arrays, where the *i*th array contains the *i*th element fro
 d3.zip([1, 2], [3, 4]); // returns [[1, 3], [2, 4]]
 ```
 
+### Blur
+
+<a name="blur" href="#blur">#</a> d3.<b>blur</b>() · [Source](https://github.com/d3/d3-array/blob/master/src/blur.js), [Examples](https://observablehq.com/@d3/d3-blur)
```

Comment:
I notice the examples link does not resolve https://observablehq.com/@d3/d3-blur

Reply:
I'll publish some examples if the PR lands

Reply:
thanks for the review curran!

Reply:
My pleasure! Thanks for your great work on this feature.